course_access_groups/views.py

```diff
@@ -33,7 +32,7 @@ def get_queryset(self):
         return self.model.objects.all()
 
 
-class MemberViewSet(CommonAuthMixin, viewsets.ModelViewSet):
+class MembershipViewSet(CommonAuthMixin, viewsets.ModelViewSet):
```

Comment:
I like this better, thanks @OmarIthawi !

Reply:
Yeah, that was a typo.